### ISSUE TITLE
onloadwff.js: Uncaught TypeError: Cannot read property 'type' of undefined

### ISSUE THREAD

**Issue by tboeckmann** [mailchain/mailchain-web#39]
**Describe the bug**
When I press Enter in the compose body, the js console in Chrome shows the following error:

```txt
onloadwff.js:71 _Uncaught TypeError: Cannot read property 'type' of undefined_
    at e.setFieldValue (onloadwff.js:71)
    at HTMLFormElement.formKeydownListener (onloadwff.js:71)
```

**To Reproduce**
1. Go to Inbox
2. Click on Compose
3. Open the JS Console
4. Type in the message body field and press Enter
5. See error

**Expected behavior**
There should be no errors

**Screenshots**
![image](https://user-images.githubusercontent.com/2022378/59126410-f318dc00-895c-11e9-87fc-cdfec6741d42.png)

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser: chrome
 - Version N/A

**Comment by tboeckmann** [comment]
This appears to be related to the Lastpass extension receiving propagated events.

We can add a workaround so extensions don't receive the enter key event info.

**Comment by tboeckmann** [comment]
fixed in #40
